pkg/env/dev.go

```diff
@@ -41,6 +41,9 @@ func newDev(ctx context.Context, log *logrus.Entry, component ServiceComponent)
 		d.features[feature] = true
 	}
 
+	// Disable applens in local dev environment: ARO-7860
+	d.Environment().PkiIssuerUrlTemplate = ""
```

Comment:
I think this should be here where the environment is created.
https://github.com/Azure/ARO-RP/blob/4c59ba73f586552561859631fc3fc648a4b8af23/pkg/util/instancemetadata/dev.go#L13